### ISSUE TITLE
Remove dependency on `browser_style: true`

### ISSUE THREAD

**Issue by tdulcet** [TinyWebEx/CommonCss#4]
[As of Firefox 118](https://bugzilla.mozilla.org/show_bug.cgi?id=1830711), support for `browser_style: true` was removed from Manifest V3. This library should include any CSS needed to follow the Firefox/Thunderbird Photon design locally. See the [migration guide on MDN](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Browser_styles#manifest_v3_migration) for more information. Also see [this comment on BMO](https://bugzilla.mozilla.org/show_bug.cgi?id=1827910#c0) for more details on what specific styles were removed and their effect.

This would be needed for https://github.com/TinyWebEx/common/issues/14.

**Comment by rugk** [comment]
Yeah AFAIK/IIRC the dependency is not large, it is mostly the style of the controls and  IIRC the style of the (reset) button was so flat that I never used it for that.

Actually I also found https://github.com/rugk/offline-qr-code/issues/44 and https://github.com/rugk/offline-qr-code/pull/97 where you can actually see the default stuff I did not want to apply.
So as it is gone now, maybe it is not too bad (though it's sad to see the cool styling of input elements being “lost” or I [again](https://github.com/rugk/awesome-emoji-picker/blob/7379aa2c559465151417621f5a044ecde55a5e67/src/popup/confirmationHint.css) need to copy Mozilla source code for that… :upside_down_face:

**Comment by Croydon** [comment]
In order to unblock migrations to MV3, I would just add the entire CSS file that `browser-style: true` injects: https://searchfox.org/mozilla-central/source/browser/components/extensions/extension.css

Any optimization like removing everything unneeded can happen somewhen in the future.

**Comment by rugk** [comment]
Hmm a neat idea, though we did not use _that_ much, IIRC. Dunno, what is your opinion, @tdulcet?

**Comment by tdulcet** [comment]
Yeah, I would vote to just add at most the small subset of global styles from the style sheet, maybe without the `user-select`:
https://github.com/TinyWebEx/CommonCss/blob/f0b844d2bbbfbd3e66e4f4dda486e3d7104f9e35/browser_style.css#L22-L40 I do not believe there are any add-ons using this library that depend on the rest. Of course, long term the library will need to be updated to better follow the Firefox/Thunderbird Photon design and unfortunately, this CSS does not seem to help much with that.
